Assets/LeapMotionModules/Interaction/ActivityManager.cs

```diff
@@ -33,38 +32,49 @@ public class ActivityManager {
       this.activationRadius = activationRadius;
     }
 
-    public void FixedUpdateHand(Hand hand) {
-      int count = GetSphereColliderResults(hand, ref _colliderResultsBuffer);
-      UpdateActiveList(count, _colliderResultsBuffer);
-    }
+    public void FixedUpdatePosition(Vector3 palmPosition, List<LeapSpace> spaces = null) {
+      using (new ProfilerSample("Update "+ (spaces==null?"Touch" : "Hover") + " Actvity Manager")) {
```

Comment:
Performing a string concat creates garbage, unfortunately, which will persist even if profiling is disabled :(